### PR TITLE
BS-13264 create a default process overview mapping if internal and empty

### PR DESCRIPTION
**Pull request by rbioteau**

### CODE REVIEW

File: bundles/plugins/org.bonitasoft.studio.pagedesigner/src/org/bonitasoft/studio/pagedesigner/core/bar/FormBarResourceProvider.java [hunk 1]
```diff
@@ -136,7 +136,7 @@ private boolean isValid(final FormMapping formMapping) {
             case URL:
                 return !isNullOrEmpty(formMapping.getUrl());
             case INTERNAL:
-                return formMapping
+                return formMappingType(formMapping) == FormMappingType.PROCESS_OVERVIEW || formMapping
```
Comment (apupier) [inline bundles/plugins/org.bonitasoft.studio.pagedesigner/src/org/bonitasoft/studio/pagedesigner/core/bar/FormBarResourceProvider.java]: FormMappingType.PROCESS_OVERVIEW.equals(..) ?

**Comment by apupier** [comment]
Don't we need also a test to check that the overview custom page is available? o rmaybe it will be tested indirectly with current Integration test? (in case this file is renamed for instance)

File: bundles/plugins/org.bonitasoft.studio.pagedesigner/src/org/bonitasoft/studio/pagedesigner/core/bar/FormBarResourceProvider.java (hunk 1 above)
Comment (rbioteau) [inline bundles/plugins/org.bonitasoft.studio.pagedesigner/src/org/bonitasoft/studio/pagedesigner/core/bar/FormBarResourceProvider.java]: it's a enum...

**Comment by rbioteau** [comment]
autogenerated overview custom page is available at runtime. It's not studio responsability.

File: bundles/plugins/org.bonitasoft.studio.pagedesigner/src/org/bonitasoft/studio/pagedesigner/core/bar/FormBarResourceProvider.java (hunk 1 above)
Comment (apupier) [inline bundles/plugins/org.bonitasoft.studio.pagedesigner/src/org/bonitasoft/studio/pagedesigner/core/bar/FormBarResourceProvider.java]: oh yes! :-)

**Comment by apupier** [comment]
"Custompage name is API" --> sure that Engine (web) guys are conscious about it?

**Comment by rbioteau** [comment]
They should, and they should have a test failing when changing this name.
I'm not aware of how it has been done.

**Pull request merged**